### PR TITLE
Add LED Pulse Control Support to Dynamic Reconfigure

### PR DESCRIPTION
**Pull request by QuentinTorg**
This PR adds LED pulse control to the dynamic reconfigure server for next gen stereo cameras that support lighting

Testing is still required

### CODE REVIEW

File: multisense_ros/cfg/multisense.cfg [hunk 1]
```diff
@@ -197,6 +197,9 @@ for cfg in sensorConfigList:
         gen.add("lighting", bool_t, 0, "Lighting", False)
         gen.add("flash", bool_t, 0, "Flash", False)
         gen.add("led_duty_cycle", double_t, 0, "DutyCycle", 0.0, 0.0, 1.0)
+        if cfg.ar0234:
+            gen.add("number_of_pulses", int_t, 0, "NumberOfPulses", 1, 0, 10)
+            gen.add("led_startup_time", int_t, 0, "LedStartupTime", 0, -10000, 10000)
```
Comment (mattalvarado) [inline multisense_ros/cfg/multisense.cfg]: Can we add a unit of time in the `led_startup_time` name?

Comment (QuentinTorg) [reply multisense_ros/cfg/multisense.cfg]: I updated to `led_startup_time_us`, and `LedStartupTimeUs`

File: multisense_ros/src/reconfigure.cpp [hunk 2]
```diff
@@ -582,6 +582,39 @@ template<class T> void Reconfigure::configureLeds(const T& dyn)
...
+        lighting::Config leds;
+
+        if (false == dyn.lighting) {
+            leds.setFlash(false);
+            leds.setDutyCycle(0.0);
+        } else {
+            leds.setFlash(dyn.flash);
+            leds.setDutyCycle(dyn.led_duty_cycle * 100.0);
+            leds.setNumberOfPulses(dyn.number_of_pulses);
+            leds.setLedStartupTime(dyn.invert_led_output ?
```
Comment (moslemk) [inline multisense_ros/src/reconfigure.cpp:601]: Shouldn't this be reversed?

**Comment by QuentinTorg** [comment]
Replaced by #48

**Pull request closed without merging**